code/datums/components/barber.dm

```diff
@@ -71,7 +71,7 @@
 		boutput(user, "<span class='notice'>You're going to need to remove that mask/helmet/glasses first.</span>")
 		non_murderous_failure = BARBERY_FAILURE
 
-	if(M.bioHolder.mobAppearance.customization_first == "None")
+	if((M.bioHolder.mobAppearance.customization_first == "None") && (M.bioHolder.mobAppearance.customization_second == "None") && (M.bioHolder.mobAppearance.customization_third = "None"))
```

Comment:
```suggestion
	if((M.bioHolder.mobAppearance.customization_first == "None") && (M.bioHolder.mobAppearance.customization_second == "None") && (M.bioHolder.mobAppearance.customization_third = "None"))
```

your last check is an assignment rather than a comparison

maybe also do like `var/datum/appearanceHolder/AH = M.bioHolder.mobAppearance` so you can make that a bit easier to read